contexts/security-v3-unstable.jsonld

```diff
@@ -1,11 +1,11 @@
 {
-  "@context": [
-    {
+  "@context": [{
       "@version": 1.1
     },
```

Comment:
```suggestion
  "@context": [{
      "@version": 1.1
    },
```

I don't think we need this `@version` stuff, let's remove it.

Reply:
@dlongley alternatively, does a top level declaration of the jsonld processor version mean we dont have to repeat it at the scoped context level?

Reply:
Term definitions in JSON-LD 1.1 are automagically upgraded to version 1.1 when 1.1 features are used -- and otherwise there's no difference in how they are processed from 1.0. This means that declaring a version is wholly unnecessary, but this wasn't figured out in the JSON-LD WG until just before 1.1 became REC -- which was *after* the VC context v1 was published.

In short, we really don't need the declaration at all. The only reason we *may* need it, is for backwards compatibility with the VC context, which was written *before* the need to use it was removed from JSON-LD 1.1. Even then, we should be able to update processors to ignore `@version` tags when comparing protected terms for equality (this would be the only reason for needing it wrt backwards compat w/VC context v1).

Also, regarding the use of scoped contexts, it's always important to remember that only what's *in scope* (what's in the scoped context) is guaranteed to be defined. So, when you use a type-scoped-context in a JSON-LD document somewhere, you need to remember that anything that isn't in its scoped-context may no longer be defined, having been removed by something further up the JSON tree. So we want to ensure we define anything that may be used within that scope.

Reply:
Sweet, removed all the usages of this term. Thanks for the insight too.